.github/workflows/extract-locales.yml

```diff
@@ -29,11 +29,15 @@ jobs:
           run: |
             make update_deps
             make extract_locales
+
+      - name: Push Lcoales (dry-run)
+        if: github.event_name == 'pull_request'
+        shell: bash
+        run: |
+          make push_locales ARGS="--dry-run"
+
       - name: Push Locales
+        if: github.event_name == 'push'
```

Comment:
we've lost the requirement that this be for the master branch along the way?

Reply:
branches:
      - master

We are already filtering on the workflow level. I could introduce it also here but it would be redundant (It was there before and was redundant there as well.)

Wdyt?

Reply:
okay, didn't check elsewhere in the file, just noticed it wasn't there any longer after your change.